spec/features/user_spec.rb

```diff
@@ -17,7 +17,7 @@
         create_card
         click_link 'Редактировать'
         fill_in :card_original_text, with: 'Another word'
-        click_button 'Создать'
+        click_button 'Обновить'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.